### PR TITLE
chore: replace absl::optional<T> with std::optional<T>

### PR DESCRIPTION
**Pull request by miniak**
Follow-up to https://chromium-review.googlesource.com/c/chromium/src/+/4469528

Notes: none

### CODE REVIEW

File: shell/browser/native_window_mac.h [hunk 1]
```diff
@@ -249,7 +249,7 @@ class NativeWindowMac : public NativeWindow,
   bool fullscreen_before_kiosk_ = false;
   bool is_kiosk_ = false;
   bool zoom_to_page_width_ = false;
-  absl::optional<gfx::Point> traffic_light_position_;
+  std::optional<gfx::Point> traffic_light_position_;
```
Comment (ckerr) [inline shell/browser/native_window_mac.h:252]: Needs an `#include <optional>` at top

**Review by ckerr — COMMENTED** [review]
There are a handful of IWYU places where `#include <optional>` needs to be added; but otherwise LGTM. :green_heart:

**Comment by miniak** [comment]
> There are a handful of IWYU places where `#include <optional>` needs to be added; but otherwise LGTM. 💚

@ckerr how do I check that next time locally?

**Comment by ckerr** [comment]
> @ckerr how do I check that next time locally?

It's always a good idea to check the headers manually whenever adding the first reference or removing the last reference to something in a file.

Upstream uses `iwyu`, I think, but I haven't looked into how they do it.

**Review by ckerr — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Pull request merged**